heat/nn/data_parallel.py

```diff
@@ -0,0 +1,275 @@
+import bisect
+import functools
+import heat as ht
+import operator
+import torch
+import torch.nn as tnn
+
+from collections import OrderedDict
+from typing import Callable, List
+
+from heat.core.communication import MPI
+
+__all__ = ["DataParallel"]
+
+
+class DataParallel(tnn.Module):
+    """
+    Implements data parallelism across multiple processes. This means that the same model will be run locally
+    on each process. Creation of the model parallels to PyTorch, the only changes are using HeAT layers (ht.nn.layer)
+    in the initialization of the network. I.E. (example code contains)
```

Comment:
ht.nn.layer as cross-reference

Reply:
there isnt something to reference here because the `ht.nn.layer` calls through to torch